web/public/index.html

```diff
@@ -33,5 +34,8 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <!-- Twitter single-event website tag code -->
+    <script src="//platform.twitter.com/oct.js" type="text/javascript"></script>
+    <script type="text/javascript">twttr.conversion.trackPid('ny03t', { tw_sale_amount: 0, tw_order_quantity: 0 });</script>
```

Comment:
Forgot, we need to change the script to `if (twttr !== undefined) { twttr.conversion... }`. Because if twitter is blocked, it doesnt load then script fails